### PR TITLE
Add Terms of Service and Privacy Policy

### PR DESCRIPTION
**Pull request by Tolga1452**

### CODE REVIEW

**Review by taynotfound — DISMISSED** [review]

File: resources/privacy-policy.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+## 1. Introduction
+
+Welcome to Experiment Hub! Experiment Hub bot ("the Bot") is designed to enhance your Discord experience by providing a collection of Discord's popular experiments and features. This Privacy Policy ("the Policy") explains how we collect, use, and protect your personal data when you use the Bot. By using the Bot, you consent to the practices described in this Policy.
+
+## 2. Data Collection
+
+2.1. **User Data**: The Bot may collect and store user data, including user IDs and server settings, to provide its services.
+
+2.2. **API Key**: To access additional AI models and features, users are required to add their own PurGPT API key from https://purgpt.xyz. We do not collect or store these API keys.
```
[the remark this answers is not in the document]
Comment (Tolga1452) [reply resources/privacy-policy.md:13]: Oops i forgot to change this one

File: resources/privacy-policy.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## 3. Data Usage
+
+3.1. **Bot Services**: The collected data is used solely to provide and improve the services offered by the Bot. This may include customizing features, enhancing user experience, and resolving technical issues.
+
+## 4. Data Protection
+
+4.1. **Security**: We are committed to protecting your data and employ reasonable security measures to prevent unauthorized access, disclosure, alteration, or destruction of your information.
+
+4.2. **Data Retention**: We will retain your data only for as long as necessary to fulfill the purposes outlined in this Privacy Policy or as required by law.
```
[the remark this answers is not in the document]
Comment (Tolga1452) [reply resources/privacy-policy.md:23]: It doesn't rn bur we can

File: resources/privacy-policy.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+6.1. The Bot is not intended for use by individuals under the age of 13. We do not knowingly collect personal information from children under 13. If you believe we have inadvertently collected such information, please contact us to have it removed.
+
+## 7. Changes to the Privacy Policy
+
+7.1. We reserve the right to update this Privacy Policy to reflect changes in our data practices. Any modifications will be posted on our [Discord server](https://discord.gg/experiments).
+
+## 8. Contact Us
+
+8.1. If you have any questions, concerns, or requests regarding your personal data or this Privacy Policy, please contact us through our Discord server.
```
Comment (Pukimaa) [inline resources/privacy-policy.md:39]:
```suggestion

## 8. Your Choices

8.1 You have the right to request data deletion. Just contact us, as highlighted in point nine.

8.2 You can choose to stop using the bot at any time by removing it from your Discord server.

## 9. Contact Us

9.1. If you have any questions, concerns, or requests regarding your personal data or this Privacy Policy, please contact us through our Discord server.
```
Let's add some Choices/Rights

**Review by Pukimaa — CHANGES_REQUESTED** [review]

File: resources/privacy-policy.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Pukimaa) [reply resources/privacy-policy.md:23]: I would just leave it like the suggestion for now, it doesn't hurt. We can always change it later, but let's be safe and get this merged quickly!

**Review by taynotfound — APPROVED** [review]

**Review by Pukimaa — APPROVED** [review]

**Review by Lordcobcob — APPROVED** [review]
There I approved it lol

**Pull request merged**